components/Footer/Components/ExternalNav.jsx

```diff
@@ -9,15 +9,15 @@ import { decode } from "html-entities";
  */
 export default function ExternalNav({ navigation, audience }) {
   return (
-    <div className="su-pb-[60px] lg:su-pb-[95px]">
+    <div className="su-pb-60 lg:su-pb-[95px]">
```

Comment:
@yvonnetangsu Any reason that the lg value isn't lg:su-pb-95 to remove the bracket notation as in other cases?

Reply:
@iamrentman No reason at all 😄 Because there are so many of these, for me to do a thorough replacement would take much longer. So my focus in this PR is to remove the deprecated classes and to use most updated conventions, e.g., use `su-shrink` and not `su-flex-shrink`. Replacing these custom classes are secondary priority in this PR since they still work. I'm thinking that if needed, we can do a final round after all the activity from Squiz dev calms down after launch. What do you think?